### ISSUE TITLE
ABI type-related errors

### ISSUE THREAD

**Issue by ConnorRepeat** [robertaachenw/solidity-debugger#4]
Hi, first of all, thanks for the elegant debug solution.

I'm trying to debug `iZiSwapPool.sol` in `https://github.com/izumiFinance/izumi-swap-core` (which is very similar to Uniswap V3, but not exactly the same) and I'm getting an error at the beginning of the debug, which seems to be that the contract provides the return value as an array type view function.

Here is the error message:

```
Unexpected solidity ABI type: tuple[] (Parameter 'name') => System.ArgumentException: Unexpected solidity ABI type: tuple[] (Parameter 'name')
   at Meadow.Core.AbiEncoding.AbiTypeMap.GetSolidityTypeInfo(String name) in /root/sdbg/SolidityDebugger/Meadow.Core/AbiEncoding/AbiTypeMap.cs:line 167
   at Meadow.SolCodeGen.CodeGenerators.ContractGenerator.GenerateOutputs(Output[] outputs) in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodeGenerators/ContractGenerator.cs:line 755
   at Meadow.SolCodeGen.CodeGenerators.ContractGenerator.GenerateFunction(Abi methodAbi) in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodeGenerators/ContractGenerator.cs:line 301
   at Meadow.SolCodeGen.CodeGenerators.ContractGenerator.GenerateClassMembers() in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodeGenerators/ContractGenerator.cs:line 136
   at Meadow.SolCodeGen.CodeGenerators.ContractGenerator.GenerateClassDef() in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodeGenerators/ContractGenerator.cs:line 82
   at Meadow.SolCodeGen.CodeGenerators.GeneratorBase.GenerateNamespaceContainer() in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodeGenerators/GeneratorBase.cs:line 49
   at Meadow.SolCodeGen.CodebaseGenerator.GeneratedContractSourceFiles(ContractInfo[] contractInfos, List`1 generatedEvents) in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodebaseGenerator.cs:line 411
   at Meadow.SolCodeGen.CodebaseGenerator.GenerateSources() in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodebaseGenerator.cs:line 334
   at Meadow.SolCodeGen.CodebaseGenerator.Generate(CommandArgs appArgs, Boolean returnFullSources, LoggerDelegate logger) in /root/sdbg/SolidityDebugger/SolidityDebugger/SolCodeGen/CodebaseGenerator.cs:line 64
   at Meadow.DebugSolSources.Program.GetSolcCompilationData(AppOptions opts) in /root/sdbg/SolidityDebugger/SolidityDebugger/Program.cs:line 192
   at Meadow.DebugSolSources.Program.Run(String[] args) in /root/sdbg/SolidityDebugger/SolidityDebugger/Program.cs:line 106
   at Meadow.DebugSolSources.Program.Main(String[] args) in /root/sdbg/SolidityDebugger/SolidityDebugger/Program.cs:line 87
```

**Comment by robertaachenw** [comment]
Thank you. I am looking into it

**Comment by robertaachenw** [comment]
Fixed in 1.0.1

**Comment by m-waqas88** [comment]
> Fixed in 1.0.1

@robertaachenw 

Really appreciate your efforts. A very brilliant and awesome tool for Security Researchers. The only thing that I am worried about is that is is built on top of an unmaintained lib.
